tests/assets/plugins/taggers/plugin_third_party_trainable_tagger.py

```diff
@@ -21,7 +21,7 @@
 
 from steamship import Block, File, SteamshipError, Tag
 from steamship.app import Response, create_handler
-from steamship.base import Task
+from steamship.base import Task, TaskState
```

Comment:
I went in and changed all the TrainableTagger classes & tests to use the new async reporting style.